integr8ly/grafana-operator
# Uid updates

Several changes and bugfixes:

* allow user defined UIDs
* centralize functions to create UIDs and Hashes for dashboards
* replace md5 hashes with sha
* remove deprecated spec (dashboard compat)
* update Grafana version to 7.1.1
* update operator-sdk to 0.18.2
* update docs

Verification steps:

1. Deploy Grafana and create example dashboards using the previous release
2. Checkout this branch and run the operator against the existing deployment
3. Make sure all dashboards are retained and that the hashes in the Grafana CR get updated
4. Verify that Grafana 7.1.1 is installed
